packages/react-widgets/src/less/select-list.less

```diff
@@ -27,7 +27,7 @@ input.rw-select-list-input {
   left: 0;
   top: 50%;
   transform: translateY(-50%);
-  top: ~'0.1em /9';
```

Comment:
this is intentional the `/9` is a hack that causes it to only be used in older IE

Reply:
Which fails if you use stylelint. IE 8 usage is fractions of a percent. If support is really needed then maybe we can generate a separate `ie8.css`?

Reply:
Honest question, why are you linting third party css? Do you also run eslint over react-widgets.js? 

Overall, I think removing existing browser coverage to appease a linter isn't a great tradeoff. Besides removing it for v4 isn't feasible as it's a breaking change. We are already bumping the min browsers for v5

Reply:
which is to say, even if they are legacy, they are the current styles and we can't just remove them without marking it as breaking!

Reply:
Sure that's fair, is there a v5 branch this can go in to?

Reply:
ya it's `v5` hopefully tho they are already gone there. I may not have gotten to that tho